development/build/static.js

```diff
@@ -80,13 +80,34 @@ const copyTargetsDev = [
     pattern: '/chromereload.js',
     dest: ``,
   },
+  // empty files to suppress missing file errors
```

Comment:
Huh, that's an interesting idea!

I had been meaning to address this warning by updating the build script to use these dependency bundles in development too, rather than just in prod. But I never got around to it. This seems like a simpler solution for now.

Reply:
yeah this solution is sort of dumb and silly but saves a good chunk of code dedicated to conditionally flipping on and off script imports. simplicity won me over here.